resources/export.py

```diff
@@ -6,20 +6,19 @@
 import os.path
 from common import *
 
-def backup_mysql_to_s3(creds, vcap):
-    access, secret, bucket, region = find_s3_bucket_creds(vcap)
-    s3_cmd = build_s3_copy_command(access, secret, region, bucket)
+# use mysqldump to backup the database and stream the result into a given store.
+def backup_mysql(creds, store_stream_cmd):
     mysql_cred = build_mysql_cred_str(creds)
-    command = 'bin/mysqldump {} | {}'.format(mysql_cred, s3_cmd)
-    os.system(command)
+    command = 'bin/mysqldump {} | {}'.format(mysql_cred, store_stream_cmd)
+    run_command(command)
 
-def backup_psql_to_s3(creds, vcap):
-    access, secret, bucket, region = find_s3_bucket_creds(vcap)
-    s3_cmd = build_s3_copy_command(access, secret, region, bucket)
-    psql_cred = build_psql_env(creds)
-    command = '{} bin/pg_dump --format=custom | {}'.format(psql_cred, s3_cmd)
-    os.system(command)
+# use pg_dump to backup the database and stream the result into a given store.
+def backup_psql(creds, store_stream_cmd):
+    psql_cred = build_psql_env(creds, 'bin/pg_dump --format=custom')
+    command = '{} | {}'.format(psql_cred, store_stream_cmd)
+    run_command(command)
 
+# for export, we need to know the bound s3 bucket. for now, just get the first bucket.
```

Comment:
Doesn't the plugin already let you choose which S3 bucket to export to or import from?  Would pulling the credentials of just the first bucket cause an issue with that or has the behavior changed as well?